src/csync/CMakeLists.txt

```diff
@@ -106,8 +98,6 @@ set_target_properties(
         ${LIBRARY_VERSION}
       SOVERSION
         ${LIBRARY_SOVERSION}
-      RUNTIME_OUTPUT_DIRECTORY
-        ${BIN_OUTPUT_DIRECTORY}
```

Comment:
Personally I don't care about this, because I always install also during development but some people might be annoyed because this changes the way executables are run from the build directory.

Reply:
I believe you can't run the client without installing it anyway because it needs to access some files from the installed directory. (or maybe this was a csync problem and it is no longer the case)

But I suppose this may also change the location of the tests. But in a way, it is better to have the tests in their directory.